resources/conf/rate-limiter-config.yaml

```diff
@@ -1,27 +1,32 @@
-# Sample config until, Adapter xDS implementation is done
 domain: default
 descriptors:
   - key: org
-    value: wso2
+    value: carbon.super
     descriptors:
       - key: vhost
         value: localhost
         descriptors:
-          # API level policy
           - key: path
+            value: /v2/ratelimitService/pet/findByStatus
```

Comment:
If this is an API-level policy (not operational level), the value should be just the basepath, without the resource paths